wow/core/src/main/scala/wow/auth/AccountsState.scala

```diff
@@ -0,0 +1,60 @@
+package wow.auth
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props, Terminated}
+import wow.Application
+import wow.auth.AccountsState.{AccountIdentifier, IsOnline, NotifyAccountOnline}
+
+import scala.collection.mutable
+
+/**
+  * Tracks online state for every account
+  */
+class AccountsState extends Actor with ActorLogging {
+  /**
+    * Map of actor ref representing account (NetworkWorker) by account identifier (login -> NetworkWorker ref)
+    */
+  private val accountRefById = new mutable.HashMap[AccountIdentifier, ActorRef]()
+
+  /**
+    * Map of account identifier by actor ref  (NetworkWorker ref -> login)
+    */
+  private val accountByActor = new mutable.HashMap[ActorRef, AccountIdentifier]()
```

Comment:
I agree on principle for the usage of the BiMap, but I do not want to pull in Guava for just that and I wanted to avoid writing a custom one since I felt it was a common enough data structure that writing it again on our own would be useless. Not sure where that leaves us, as I'm not really happy with pulling all of Guava, but I'm not sure where to find a good bimap otherwise.